content/registry/recipes/mirror.md

```diff
@@ -101,9 +101,12 @@ proxy:
 > made available on your mirror. **You must secure your mirror** by
```

Comment:
Looks like this one has a display issue in the [preview](https://deploy-preview-17988--docsdocker.netlify.app/registry/recipes/mirror/#configure-the-cache) too.